apps/electron-app/src/main/services/gmail-service.ts

```
@@ -136,139 +144,494 @@ export class GmailOAuthService {
+      logger.error("[GmailAuth] Failed to create OAuth view:", error);
+      throw error;
+    }
+  }
+
+  /** Start callback server for OAuth flow */
+  private async startSecureCallbackServer(
+    viewManager: ViewManagerState,
+    currentWindow?: BrowserWindow,
+  ): Promise<void> {
+    // Check if server is already running
+    if (this.server && this.server.listening) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)